server/app/views/components/Icons.java

```diff
@@ -182,7 +285,10 @@ public static ContainerTag svg(String pathString, int pixelSize) {
   }
 
   public static ContainerTag svg(String pathString, int width, int height) {
-    return svg(pathString).attr("viewBox", String.format("0 0 %1$d %2$d", width, height));
+    return svg(pathString)
+        .attr("viewBox", String.format("0 0 %1$d %2$d", width, height))
```

Comment:
Setting the viewBox to a specific height/width is insufficient to actually cause the SVG's bounds to match. Here, I explicitly set width / height of the SVG element, which is more consistent with what one would expect based on the method / argument names.

Reply:
Maybe leave a comment to that effect? If you need to explain it to us here, others reading the code might have the same question

Reply:
Maybe leave a comment to that effect? If you need to explain it to us here, others reading the code might have the same question

Reply:
Done.